python/lsst/analysis/tools/actions/plot/propertyMapPlot.py

```diff
@@ -0,0 +1,443 @@
+                        values[zidx],
+                        bins=bins,
+                        label=histLabel,
+                        color=color,
+                        weights=weights,
+                        alpha=0.6,
+                    )
+
+            # Set labels and legend.
+            ax2.set_xlabel(label)
+            ax2.set_ylabel("Normalized Count")
+            legend = ax2.legend(loc="best", frameon=False, fontsize=15)
+            for line, text in zip(legend.legendHandles, legend.get_texts()):
+                text.set_color(line.get_facecolor())
+
+            # Add nside info to `plotInfo`.
+            plotInfo["nside_coverage"] = mapData.nside_coverage
+            plotInfo["nside_sparse"] = mapData.nside_sparse
+            plotInfo["valid_area"] = mapData.get_valid_area()
+
+            # Add useful information to the plot.
+            figDict[mapName] = self.addPlotInfo(fig, plotInfo, mapName)
+
+            _LOG.info(
+                f"Made property map plot for dataset type {mapName}, tract: {plotInfo['tract']}, "
+                f"band: '{plotInfo['band']}'."
+            )
+
+        return figDict
+
+    def getOutputNames(self, config=None) -> Iterable[str]:
+        # Docstring inherited.
+
+        # Names needed for making corresponding output connections for the maps
+        # that are configured to run.
+        outputNames: tuple[str] = ()
+        for propertyName in config.properties:
+            coaddName = config.properties[propertyName].coaddName
+            for operationName in config.properties[propertyName].operations:
+                outputNames += (f"{coaddName}Coadd_{propertyName}_{operationName}",)
+
+        return outputNames
+
+    @staticmethod
+    def getZoomedExtent(fullExtent, n):
+        """Get zoomed extent centered on the original full plot.
+
+        Parameters
+        ----------
+        fullExtent : `tuple` of `float`
+            The full extent defined by (lon_min, lon_max, lat_min, lat_max):
+
+            lon_min : `float`
+                Minimum longitude of the original extent.
+
+            lon_max : `float`
+                Maximum longitude of the original extent.
+
+            lat_min : `float`
+                Minimum latitude of the original extent.
+
+            lat_max : `float`
+                Maximum latitude of the original extent.
+
+        n : `float`, optional
+            Zoom factor; for instance, n=2 means zooming in 2 times at the
+            center. If None, the function returns None.
+
+        Returns
+        -------
+        Results : `tuple` [`float`]
+            New extent as (new_lon_min, new_lon_max, new_lat_min, new_lat_max).
+        """
+        if n is None:
+            return None
+        lon_min, lon_max, lat_min, lat_max = fullExtent
+        lon_center, lat_center = (lon_min + lon_max) / 2, (lat_min + lat_max) / 2
+        half_lon = (lon_max - lon_min) / (2 * n)
+        half_lat = (lat_max - lat_min) / (2 * n)
```

Comment:
When doing the zoom, you want to make sure you scale the longitude (ra) range by cos(lat) so it works at a wide range of latitudes/declinations.

Reply:
However, I'm uncertain if it might lead to issues in particular scenarios, noticeably altering the aspect ratios of the zoomed-in plots compared to the full-tract plot.

Reply:
I'm pretty sure that _not_ doing this scaling will mess up the aspect ratio.  The full tract is defined in terms of degrees and will have the scaling naturally included.  So that's why we need it here for the zoom as well.